client/sites-dashboard/components/sites-ellipsis-menu.tsx

```diff
@@ -450,7 +473,11 @@ export const SitesEllipsisMenu = ( {
 						<CopySiteItem { ...props } onClick={ startSiteCopy } />
 					) }
 					<MenuItemLink
-						href={ `/settings/performance/${ site.slug }` }
+						href={
+							isWpAdminInterface
```

Comment:
Similarly, I think we can create a `getPerformanceUrl()` or something and put the logic there (to follow the style)